settings.py

```diff
@@ -1,5 +1,6 @@
 # The OSF website URL, and the API
-base_urls = ['https://osf.io/', 'https://api.osf.io/v2/']
+# base_urls = ['https://osf.io/', 'https://api.osf.io/v2/']
+base_urls = ['http://localhost/', 'http://localhost:8000/v2/']
```

Comment:
The better practice is to have a default_settings.py and a locals.py so that we don't keep track to locals.py.